src/OrchardCore.Modules/OrchardCore.Html/Migrations.cs

```diff
@@ -40,13 +40,15 @@ public int Create()
         }
 
         // This code can be removed in a later version.
+#pragma warning disable CA1822 // Mark members as static
```

Comment:
We will remove some steps in the future, but for now in case someone having e.g. the version 2 and want to upgrade.

But not the goal of this PR, it is mainly to fix analyzer rules